src/enum/call.directions.ts

```diff
@@ -0,0 +1,4 @@
+export const CALL_DIRECTIONS = {
```

Comment:
TypeScript has a great feature called enums, we haven't used them much, but let's change that! https://www.typescriptlang.org/docs/handbook/enums.html